pkg/opssight/prometheus.go

```diff
@@ -139,8 +139,8 @@ func (p *SpecConfig) PerceptorMetricsService() (*components.Service, error) {
 	})
 
 	err := service.AddPort(horizonapi.ServicePortConfig{
-		Port:       9090,
-		TargetPort: "9090",
+		Port:       3006,
```

Comment:
Why is this changing from 9090 to 3006?

Reply:
Someplace it is 3006 and in other place it is 9090. so making it consistent